cmd/commands/deploy.go

```diff
@@ -109,10 +109,12 @@ func deployAction(ctx context.Context, a inngest.ActionVersion) error {
 		// an up-to-date image and checksum for the action.
 		ui, err := cli.NewBuilder(ctx, cli.BuilderUIOpts{
 			QuitOnComplete: true,
-			BuildOpts: dockerdriver.BuildOpts{
-				Path:     ".",
-				Tag:      tag,
-				Platform: "linux/amd64",
+			BuildOpts: []dockerdriver.BuildOpts{
+				dockerdriver.BuildOpts{
+					Path:     ".",
```

Comment:
Not needed for this PR, but when we update the deploy cmd for multi-function deploys or deploying a specific directory, we could pass the path through to this :)